content/doc/addons/metabase.md

```diff
@@ -0,0 +1,181 @@
+
+- Navigate and **explore data** in the connected databases
+- Create **dashboards** that regroup multiple questions in a canva
+- **Integrate** questions/dashboards in another application
+- **Share** a questions/dashboard to anyone using an anonymous link
+- **Periodically send** results of a question/dashboard by email/Slack
+- Configure **data visualization** for questions results: table, line chart, pie chart, gauge, single number, etc.
+- Connect **external databases**; it works with add-ons such as [MySQL](/doc/addons/mysql/), [PostgreSQL](/doc/addons/postgresql/), [MongoDB](/doc/addons/mongodb/) and [many other data sources](https://www.metabase.com/data_sources/)
+- Create _questions_ either by typing in `SELECT` SQL queries or by using Metabase’s UI to build such queries **without using SQL**
+
+You can also save questions and **organize** them in _collections_. When opening a saved question, fresh data is extracted from the source DB, so that questions always show fresh results. You get user management with **groups and permissions**: users can access to whole data sources so that they can explore and create questions, or they can have access only to collections containing already existing questions/dashboard.
+
+## Create a Metabase add-on
+
+### From the Console
+
+1. [Create a new add-on](https://console.clever-cloud.com/users/me/addons/new) by clicking the **Create...** dropdown in the sidebar and then **an add-on**
+2. Select the Metabase add-on
+3. You can skip linking the add-on to an application, it won't be needed (connecting data sources to Metabase is done via Metabase's UI)
+4. Enter a name for your Metabase add-on and select the zone where you want it to be deployed
+
+### Using the CLI
+
+1. Make sure you have `clever-tools` installed locally. Please refer to the [setup guide](https://github.com/CleverCloud/clever-tools/blob/master/docs/setup-systems.md) if needed
+2. List the available plans and options for Metabase: `clever addon providers show metabase`
+3. In your terminal, run `clever addon create metabase <name> --org <org>` (`--org` is optional)
+
+Refer to the [Clever Tools documentation](https://github.com/CleverCloud/clever-tools/blob/master/docs/addons-backups.md) for more details on add-on management.
+
+## Acessing the Metabase Interface
+
+Once deployed, Metabase is accessible through an URL provided in the add-on's `Service dependencies` page. You can also go in the environment variables page of the Java application and look for `MB_SITE_URL` value. To get this URL with Clever Tools, use the add-on ID:
+
+```
+$ clever addon env ADDON_ID
+METABASE_URL="https://an_id-metabase.services.clever-cloud.com"
```

Comment:
Make sure that this URL makes sense (cc @allemas)

Reply:
It's the env var as sent back by the add-on once created. As we planned to modify it, this change LGTM